src/draw/grouping.typ

```diff
@@ -106,9 +138,10 @@
 /// The default anchor is "center" but this can be overridden by using `anchor` to place a new anchor called "default".
 ///
 /// - body (elements, function): Elements to group together. A least one is required. A function that accepts `ctx` and returns elements is also accepted.
-/// - name (none, string):
-/// - anchor (none, string):
-/// - ..style (style):
+/// - name (none, string): Name of the group element
+/// - anchor (none, string): Anchor to position the group and it's children relative to. For translation the difference between the groups `"center"` anchor
+///   and the passed anchor is used.
+/// - ..style (style): Style key-value pairs
```

Comment:
I haven't been documenting these arguments as they always act the same on elements and are documented elsewhere. They still show up on the function definition even if they are not written here. 

Maybe the anchor description is okay as its behaviour should be noted but it could be moved to under the anchor heading.

Reply:
Reverted. I left the anchor description, though.